packages/block-library/src/query/deprecated.js

```diff
@@ -38,6 +43,12 @@ const deprecated = [
 		supports: {
 			html: false,
 		},
+		migrate( attributes ) {
```

Comment:
@ntsekouras I noticed that when opening the site editor (tt1-blocks), the query block gets a `{ type: 'list' }` layout which I think is meant to be in `displayLayout` right now? so I added this migration, I'm not 💯 % certain that this is correct.

Reply:
Good catch! Thanks!